kokoro/gcp_ubuntu_ggp/nightly.cfg

```diff
@@ -3,7 +3,7 @@
 
 # Location of the bash script. Should have value <github_scm.name>/<path_from_repository_root>.
 # github_scm.name is specified in the job configuration (next section).
-build_file: "orbitprofiler/kokoro/gcp_ubuntu_ggp/kokoro_build.sh"
+build_file: "orbitprofiler/kokoro/gcp_ubuntu_ggp/kokoro_release_build.sh"
```

Comment:
This change is obsolete, isn't it?

Reply:
Yes, thank you. Just commited a new version